modules/bound-sa-tokens-configuring.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * authentication/bound-service-account-tokens.adoc
+
+[id="bound-sa-tokens-configuring_{context}"]
+= Configuring bound service account tokens using volume projection
+
+You can configure Pods to request bound service account tokens by using volume projection.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Set the service account issuer.
+
+.. Edit the `cluster` authentication object.
++
+----
+$ oc edit authentications cluster
+----
+
+.. Set the `spec.serviceAccountIssuer` field to the desired service account issuer value. The default is [x-]`https://kubernetes.default.svc`.
++
+[source,yaml]
+----
+spec:
+  serviceAccountIssuer: https://test.default.svc
+----
++
+// TODO: Does this have to start with `https://`?
++
+// TODO: does this value relate to the pod spec below at all?
```

Comment:
(No action required) The pod spec should set the audience it expects to provide the token to. If the pod wants to present the token to the apiserver, then it needs to either set the issuer as the audience or leave it blank (since the default is the issuer). It appears that this is already documented in `<3>`.